administrator/components/com_media/tmpl/plugin/default.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
it can be used by 3dps to close any new window opened by themselves with a single line of code